django/template/context.py

```diff
@@ -7,6 +7,7 @@
 
 class ContextPopException(Exception):
     "pop() has been called more times than push()"
+
     pass
```

Comment:
This one is weird but given `pass` is completely unnecessary in the first place I guess that's to be expected.